ci-operator/step-registry/baremetalds/e2e/ovn/bgp/pre/baremetalds-e2e-ovn-bgp-pre-commands.sh

```diff
@@ -201,6 +201,34 @@ deploy_agnhost_container() {
   $CLI exec $name ip -6 route add default dev eth0 via 2001:db8:2::2
 }
 
+gateway_mode=$(oc get networks.operator.openshift.io cluster -o jsonpath='{.spec.defaultNetwork.ovnKubernetesConfig.gatewayConfig.routingViaHost}')
```

Comment:
Could you add a reference to this beeing required due to  https://issues.redhat.com/browse/OCPBUGS-42993